src/components/episode.css

```diff
@@ -101,10 +101,60 @@ progress::-moz-progress-bar {
 }
 
 .audio-control {
-  background: none;
-  border: none;
+  border: 2px solid #f05657;
   cursor: pointer;
   margin-right: 24px;
+  outline: none;
+  background: white;
+  height: 65px;
+  width: 65px;
+  border-radius: 50%;
+  position: relative;
+  box-shadow: 4px 4px rgba(255,255,255,.5);
+  transition: all .1s ease-in;
```

Comment:
Can we refactor to remove `all` as it's not as performant as listing the individual animatable properties :) Thanks!

Reply:
Totally agree. I was just copying what the site already had. haha.

Reply:
Oh shit hahah well.... 😏 I will yell at myself then hehe